vnext/Shared/BaseScriptStoreImpl.cpp

```diff
@@ -147,7 +147,11 @@ std::unique_ptr<const jsi::Buffer> LocalFileSimpleBufferStore::getBuffer(const s
   }
 
   if (Microsoft::React::GetRuntimeOptionBool("JSI.MemoryMappedScriptStore")) {
-    return Microsoft::JSI::MakeMemoryMappedBuffer(winrt::to_hstring(storeDirectory_ + bufferId).c_str());
+    try {
+      return Microsoft::JSI::MakeMemoryMappedBuffer(winrt::to_hstring(storeDirectory_ + bufferId).c_str());
+    } catch (const facebook::jsi::JSINativeException &e) {
```

Comment:
>&e [](http://example.com/codeflow?start=53&length=2)

Should there be any telemetry about this error? Similar to ShipAssert?

Reply:
The non-memory-mapped approach doesn't include such telemetry, so don't know if this is the ideal spot for such counts/measurements.

Also, I don't think we can do shipasserts in RNW (non-Office) code.